### PR TITLE
libdef: Axios v0.13.x Updates and Improvements

### PR DESCRIPTION
**Pull request by bassettsj**
1. Updated the library defition to include all the [configuration options](https://github.com/mzabriskie/axios#request-config).
2. Added the [actual exports](https://github.com/mzabriskie/axios/blob/master/lib/axios.js#L1).

### CODE REVIEW

**Comment by thejameskyle** [comment]
cc @mzabriskie

File: definitions/npm/axios_v0.13.x/test_axios-v0.13.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+(axios.post('bar', {}, {
+  headers: {
+    foo: 'asdf',
+  },
+  xsrfCookieName: 'cookie',
+}): Promise<*>);
+// $ExpectError
+(axios.post(123): Promise<*>)
+
+const client = axios.create();
```
Comment (thejameskyle) [inline definitions/npm/axios_v0.13.x/test_axios-v0.13.js]: Want to add some more of the examples?

```js
// Send a POST request
axios({
  method: 'post',
  url: '/user/12345',
  data: {
    firstName: 'Fred',
    lastName: 'Flintstone'
  }
});
```

```js
// Send a GET request (default method)
axios('/user/12345');
```

Comment (bassettsj) [inline definitions/npm/axios_v0.13.x/test_axios-v0.13.js]: Yup, I was meaning to add actual amount of tests.

**Comment by bassettsj** [comment]
@thejameskyle I added a few more tests

**Comment by marudor** [comment]
Would favor a merge now.

**Pull request merged**